batch/batch/front_end/templates/job.html

```diff
@@ -210,16 +217,16 @@ <h2>Job Environment Variables</h2>
   </thead>
   {% if job_specification %}
   <tbody>
-  {% for envvar in job_specification['env'] %}
+    {% for envvar in job_specification['env'] %}
     <tr>
       <td>{{ envvar['name'] }}</td>
       <td>{{ envvar['value'] }}</td>
     </tr>
-  {% endfor %}
+    {% endfor %}
   </tbody>
   {% endif %}
 </table>
 
 <h2>Status</h2>
 <pre>{{ job_status_str }}</pre>
-{% endblock %}
+{% endblock %}
```

Comment:
New line at the end here.

Reply:
When I try adding it and save it, it removes it.